packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissible.ts

```diff
@@ -21,23 +20,55 @@ export function useInternalChipDismissible({
   const availableChipOptions = chipOptions.filter(
     chip => !selected.includes(chip.value),
   );
-  const { liveAnnounce } = useLiveAnnounce();
 
   const actions = {
-    handleChipRemove: (chip: typeof chipOptions[number]) => {
-      return () => {
-        liveAnnounce(`${chip.label} Removed`);
-        onChange(selected.filter(val => val !== chip.value));
-      };
+    handleChipRemove: (value: string) => {
+      return () => onChange(selected.filter(val => val !== value));
     },
+
     handleChipAdd: (value: string) => onChange([...selected, value]),
     handleCustomAdd: onCustomAdd,
+
     handleChipClick: (value: string) => {
       if (onClick === undefined) return;
       return (event: MouseEvent<HTMLButtonElement>) => onClick(event, value);
     },
+
     handleEmptyBackspace: () => {
-      actions.handleChipRemove(sortedVisibleChipOptions[selected.length - 1])();
+      actions.handleChipRemove(selected[selected.length - 1])();
+    },
+
+    handleWrapperKeyDown: (event: KeyboardEvent<HTMLDivElement>) => {
+      const target = event.target;
+      if (!(target instanceof HTMLElement)) return;
+      if (target instanceof HTMLInputElement && target.value) return;
+      if (
+        event.key === "ArrowLeft" &&
+        target.previousElementSibling instanceof HTMLElement
+      ) {
+        target.previousElementSibling.focus();
+      }
+      if (
+        event.key === "ArrowRight" &&
+        target.nextElementSibling instanceof HTMLElement
+      ) {
+        target.nextElementSibling.focus();
+      }
+    },
+
+    handleChipKeyDown: (value: string) => {
+      return (event: KeyboardEvent<HTMLDivElement>) => {
+        if (event.key === "Backspace") {
+          const target = event.target;
+          if (
+            target instanceof HTMLElement &&
+            target.nextElementSibling instanceof HTMLElement
```

Comment:
why are we doing nextElementSibling?
I think this is a good place to do some documentation through code.
Like set a variable
```
const elementWeNeedToFocusCauseBla = target.nextElementSibling

```